learn/data_backup/snapshots_vs_dumps.mdx

```diff
@@ -1,37 +1,38 @@
 ---
-title: Snapshots vs dumps — Meilisearch documentation
+title: Snapshots and dumps — Meilisearch documentation
 description: "Meilisearch offers two types of backups: snapshots and dumps. Snapshots are mainly intended as a safeguard, while dumps are useful when migrating Meilisearch."
 ---
 
-# Data backup: Snapshots vs dumps
+# Snapshots and dumps
 
-Meilisearch has two ways to backup its data: `snapshots` and `dumps`.
+This article explains Meilisearch's two backup methods: snapshots and dumps.
 
 ## Snapshots
 
-Snapshots make it possible to schedule the creation of hard copies of your database. These copies are bound to a specific Meilisearch version.
+A snapshot is an exact copy of the Meilisearch database, located by default in `./data.ms`. [Use snapshots for quick and efficient backups of your instance](/learn/advanced/snapshots).
 
-This feature is **intended mainly as a safeguard**: if something goes wrong in an instance, you're able to relaunch your database quickly and efficiently.
+The documents in a snapshot are already indexed and ready to go, greatly increasing import speed. However, snapshots are not compatible between different versions of Meilisearch.
 
-The documents in a snapshot are already indexed and ready to go, greatly increasing import speed. However, **snapshots are not compatible between different versions of Meilisearch**.
+In short, snapshots are safeguard: if something goes wrong in an instance, you're able to recover and relaunch your database quickly. You can also schedule periodic snapshot creation.
```

Comment:
Here, we could also indicate that snapshots are much bigger than dumps in general.


People who want to share a database between multiple data centers like to know this kind of info since bandwidth is usually one of the most expensive things, and they would happily trade CPU time vs bandwidth and network time.